sos/report/plugins/auditd.py

```diff
@@ -33,9 +33,20 @@ def setup(self):
             "auditctl -l"
         ])
 
+        config_file = "/etc/audit/auditd.conf"
+        log_file = "/var/log/audit/audit.log"
+        try:
+            for line in open(config_file).read().splitlines():
+                words = line.split('=')
+                if words[0].strip() == 'log_file':
```

Comment:
I think this may have issues on empty lines. We'll want to either wrap this in a try block, or put some other safeguard against empty lines in this loop.

Reply:
I've done some tests for the empty lines and so far I haven't got any backtrace. It looks like read() manages well these lines. But just to be on the safe side, I can add a safeguard. What about a check that word is not '' before splitting? Would that work?

Reply:
Either `if not line: continue` or a try/except block would be fine.